spec/lib/fingerprint_self_test_spec.rb

```diff
@@ -21,6 +21,13 @@
         expect(db.match_key).not_to be_empty
       end
 
+      it "has valid 'preference' value" do
+          # Reserve values below 0.10 and above 0.90 for users
+          # See xml/fingerprints.xsd
+          expect(db.preference.class).to be ::Float
+          expect(db.preference).to be_between(0.10, 0.90)
```

Comment:
Yeah, fair point.  Another side effect of this is that if we *don't* allow things in the `xml` directory with values outside of this range, they won't be able to be tested in the same way that the shipped versions can.  For example, the specs ensure that the examples match and that the ordering is correct.  The utilities in `bin` don't do that.  My be no good solution to this right now. Your call.